### ISSUE TITLE
[🐛 Bug]: 'capabilities' property missing on Options.Testrunner type since v9

### ISSUE THREAD

**Issue by gavvvr** [webdriverio/webdriverio#13769]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

v9.0.0+

### Node.js Version

v22.6.0

### Mode

Standalone Mode

### Which capabilities are you using?

```typescript
{
  browserName: 'chrome'
}
```

### What happened?

VSCode complains that there is no `capabilities` property on `Options.Testrunner` type in `wdio.conf.ts`

<img width="782" alt="image" src="https://github.com/user-attachments/assets/0f43c7e9-2eab-4ba3-a53b-e61bcada2819">

### How to reproduce the bug.

Simply init a e2e browser test (`npm init wdio@latest .`) and choose TypeScript

### Relevant log output

```typescript
Object literal may only specify known properties, and 'capabilities' does not exist in type 'Testrunner'.ts(2353)
```
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by gavvvr** [comment]
The `capabilities` property got removed in #12987

**Comment by danielhjacobs** [comment]
Yeah, I was wondering about this. In https://github.com/ruffle-rs/ruffle/pull/17539/files#diff-43876b7e587c2411967305f67613b324ff80223e6722ed37698884d37e176c06R212 we were switching the config to be type-less because no type was working.

**Comment by christian-bromann** [comment]
@gavvvr thanks for raising the issue. With v9 forwards we suggest users to define their config like this:

```ts
export const config: WebdriverIO.Config = {
  // ...
}
```

This was a breaking change that we should better document. Let me rename the issue.

**Comment by Delta456** [comment]
Hi, I will be taking up this issue.
